tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -74,7 +74,14 @@ class TabularPredictor(TabularPredictorV1):
         Verbosity levels range from 0 to 4 and control how much information is printed.
         Higher levels correspond to more detailed print statements (you can set verbosity = 0 to suppress warnings).
         If using logging, you can alternatively control amount of information printed via `logger.setLevel(L)`,
-        where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
+        where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels).
+    sample_weight : str, default = None
+        If specified, this column-name indicates which column of the data should be treated as sample weights. This column will NOT be considered as a predictive feature.
+        Sample weights should be non-negative (and cannot be nan), with larger values indicating which rows are more important than others.
```

Comment:
Should we suggest that the user make the values sum to 1 as a best practice? Otherwise they could get inconsistencies using the sample_weight outside of predictor.